assets/src/edit-story/elements/image/scalePanel.js

```diff
@@ -46,8 +46,8 @@ const Container = styled.div`
 	width: ${ ( { width } ) => `${ Math.max( width, MIN_WIDTH ) }px` };
 	height: ${ HEIGHT }px;
 
-	background: ${ ( { theme } ) => theme.colors.bg.v7 };
-	border-radius: 4px;
+	background: ${ ( { theme } ) => theme.colors.bg.v0 + 'EE' };
```

Comment:
Hmm. I'm not a big fun of this. What if the color is defined via `rgba()`? Or already has transparency? Let's just bring it in the theme as a separate constant for now.